components/financial_assistance/app/controllers/financial_assistance/applicants_controller.rb

```diff
@@ -222,6 +222,8 @@ def applicant_parameters
         { :emails_attributes => [:kind, :address, :id, :_destroy],
           :ethnicity => [], :immigration_doc_statuses => [] }
       ]
+      result_hash.select {|key| key.is_a?(Hash) && key[:addresses_attributes]}.first[:addresses_attributes] << :county if EnrollRegistry.feature_enabled?(:display_county)
```

Comment:
The condition fails if the select output is nil. 
- Update the logic to handle nil value.
- Rename the variable name(**result_hash**)